3-enrich/scala-common-enrich/.scalafmt.conf

```diff
@@ -0,0 +1,12 @@
+align = most
```

Comment:
let's go with some, that'll remove those weird alignments @chuwy was talking about

Reply:
also can we have `style = default` at the top?